toqito/helper/bell_notation_conversions.py

```diff
@@ -0,0 +1,748 @@
+                    - sum_b_marg
+                    + sum_ab_joint
+                )
+
+    return v_mat
+
+
+def fc_to_fp(fc_mat: np.ndarray, behaviour: bool = False) -> np.ndarray:
+    r"""Convert a Bell functional or behaviour from Full Correlator (FC) to Full Probability (FP) notation.
+
+    Assumes binary outcomes (oa=2, ob=2) corresponding to physical values +1 and -1.
+    The FP tensor indices `a, b = 0, 1` correspond to outcomes `+1, -1` respectively.
+
+    The Full Correlator (FC) notation is represented by a matrix (see :func:`fc_to_cg`).
+    The Full Probability (FP) notation represents the full probability distribution
+    :math:`V(a, b, x, y) = P(\text{out}_A=a', \text{out}_B=b' | x, y)`,
+    where :math:`a=0 \rightarrow a'=+1`, :math:`a=1 \rightarrow a'=-1` (similarly for b),
+    stored as a 4D numpy array `V[a, b, x, y]`.
+
+    This function converts from FC to FP notation.
+
+    Examples
+    ==========
+
+    Consider the CHSH inequality functional in FC notation:
+
+    .. math::
+        \text{CHSH}_{FC} =
+        \begin{pmatrix}
+            0 & 0 & 0 \\
+            0 & 1/4 & -1/4 \\
+            0 & -1/4 & 1/4
+        \end{pmatrix}
+
+    Converting to FP notation:
+
+    >>> import numpy as np
+    >>> from toqito.helper import fc_to_fp
+    >>> chsh_fc = np.array([[0, 0, 0], [0, 0.25, -0.25], [0, -0.25, 0.25]])
+    >>> fc_to_fp(chsh_fc) # doctest: +NORMALIZE_WHITESPACE
+    array([[[[ 0.25, -0.25],
+             [-0.25,  0.25]],
+            [[-0.25,  0.25],
+             [ 0.25, -0.25]]],
+           [[[-0.25,  0.25],
+             [ 0.25, -0.25]],
+            [[ 0.25, -0.25],
+             [-0.25,  0.25]]]])
+
+    Consider a behaviour (correlation matrix) in FC notation (e.g., from PR box):
+    Note: This FC matrix corresponds to the PR box *after* applying `fp_to_fc(pr_box, behaviour=True)`,
+    which uses the QETLAB convention of averaging marginal correlators.
+
+    .. math::
+        P_{FC} =
+        \begin{pmatrix}
+            1 & 0 & 0 \\
+            0 & 1/\sqrt{2} & 1/\sqrt{2} \\
+            0 & 1/\sqrt{2} & -1/\sqrt{2}
+        \end{pmatrix}
+
+    Converting to FP notation:
+
+    >>> p_fc = np.array([[1, 0, 0], [0, 1/np.sqrt(2), 1/np.sqrt(2)], [0, 1/np.sqrt(2), -1/np.sqrt(2)]])
+    >>> fc_to_fp(p_fc, behaviour=True) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+    array([[[[0.4267767, 0.4267767],
+             [0.4267767, 0.0732233]],
+    <BLANKLINE>
+            [[0.0732233, 0.0732233],
+             [0.0732233, 0.4267767]]],
+    <BLANKLINE>
+    <BLANKLINE>
+           [[[0.0732233, 0.0732233],
+             [0.0732233, 0.4267767]],
+    <BLANKLINE>
+            [[0.4267767, 0.4267767],
+             [0.4267767, 0.0732233]]]])
+
+    :param fc_mat: The matrix in Full Correlator notation.
+    :param behaviour: If True, assume input is a behaviour (default: False, assume functional).
+    :return: The probability tensor V[a, b, x, y] in Full Probability notation (oa=2, ob=2).
```

Comment:
```suggestion
    :return: The probability tensor :math:`V[a, b, x, y]` in Full Probability notation (oa=2, ob=2).
```
make sure you use inline math mode properly. Sometimes you use ` and sometimes you use nothing.